python2verilog/ir/context.py

```diff
@@ -250,7 +250,7 @@ def default_output_vars(self):
         """
         Sets own output vars to default based on number of output variables
         """
-        assert self.output_types is not None
+        assert self.output_types is not None, f"{self.name}"
```

Comment:
Make this more descriptive or remove

Reply:
I can't think of a way to make it more descriptive. I think removing it will make it less descriptive?

Reply:
I would suggest something like `f"{self.name}.output_types is valid` or something. I'm unsure how the asserts look though.

Reply:
Python does a good job of reporting assertion failures. It looks like
![image](https://github.com/WorldofKerry/ToHDL/assets/43691721/8f9ab95e-672e-4045-800c-0a22096c4182)
which I think is pretty self-documenting. I also don't think this function can define that `None` means not valid.